packages/odata-service-inquirer/src/translations/odata-service-inquirer.i18n.json

```
@@ -229,7 +221,16 @@
     },
     "warnings": {
-        "largeMetadataDocument": "The metadata for this OData service is significantly large. It may take some time before this operation completes."
+        "largeMetadataDocument": "The metadata for this OData service is significantly large. It may take some time before this operation completes.",
+        "warningCertificateValidationDisabled": "User has disabled certificate validation",
+        "nonUIServiceTypeWarningMessage": "Please note that {{serviceType}} services, or not classified services, are not intended to be used for the generation of SAP Fiori UI applications",
+        "noServicesAvailable": "No services available for the selected system, see logs for further details.",
+        "noServicesAvailableForOdataVersion": "There are no V{{odataVersion}} OData services available from the selected system and the template you have chosen supports V{{odataVersion}} OData services only",
+        "noAnnotations": "No backend annotations associated with this service were retrieved and may result in an invalid application being created",
+        "collaborativeDraftMessage": "The selected service is draft enabled but does not support collaborative draft.",
+        "certificateErrors": "A certificate error is being returned when connecting to host: {{-url}}. Certificate error: {{error}}",
+        "allowingUnauthorizedCertsNode": "Setting the environment variable 'NODE_TLS_REJECT_UNAUTHORIZED' to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification. It is important to understand the security risks when using this setting.",
+        "certErrorIgnoredByNodeSetting": "An SSL certificate error is being ignored by the node setting 'NODE_TLS_REJECT_UNAUTHORIZED=0'. This makes TLS connections and HTTPS requests insecure by disabling certificate verification. See logs for the certificate error details."
```

```suggestion
        "certErrorIgnoredByNodeSetting": "An SSL certificate error is being ignored by the `NODE_TLS_REJECT_UNAUTHORIZED=0` environment variable. This makes TLS connections and HTTPS requests insecure by disabling certificate verification. For more information, view the logs."
```

---

@lfindlaysap This doesnt seem technically correct to me. The variable is `NODE_TLS_REJECT_UNAUTHORIZED` and the value is '0' hence the statement ""An SSL certificate error is being ignored by the node setting 'NODE_TLS_REJECT_UNAUTHORIZED=0'"

---

I assumed since we were mentioning the same variable in https://github.com/SAP/open-ux-tools/pull/3265/files#diff-0fc31f4893572c970fa4c2e9ea8341dc76433e27321c459fc24a9765e351113dR232 that we should refer to them in the same way. Is this occurrence in a different context than above?